### PR TITLE
adding new Bing Image Search quickstart

### PR DESCRIPTION
**Pull request by aahill**
## Purpose

* ...
This is a new quickstart code sample to compliment upcoming documentation changes on docs.microsoft.com
## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [https://github.com/aahill/cognitive-services-java-sdk-samples.git]
cd [cognitive-services-java-sdk-samples/Search/BingImageSearch/Quickstart]
git checkout [new-bing-image-quickstart]
run the java sample
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...
That the application prints out a URL from the Bing Image Search API
## Other Information

You will need to sign up for Azure cognitive services and use a subscription key.

### CODE REVIEW

File: Search/BingImageSearch/Quickstart/.factorypath [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<factorypath>
```
Comment (milismsft) [inline Search/BingImageSearch/Quickstart/.factorypath]: Can you please remove this file? I don't think it's part of the sample.

File: Search/BingImageSearch/Quickstart/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (milismsft) [inline Search/BingImageSearch/Quickstart/.vscode/launch.json]: Same for this one

File: Search/BingImageSearch/multiFeatured/.factorypath [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<factorypath>
```
Comment (milismsft) [inline Search/BingImageSearch/multiFeatured/.factorypath]: And this one

File: Search/BingImageSearch/multiFeatured/.vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (milismsft) [inline Search/BingImageSearch/multiFeatured/.vscode/launch.json]: and this one

File: Search/BingImageSearch/multiFeatured/.gitignore [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+.settings/
+.loadpath
+
+# Other Tooling #
+.classpath
+.project
+target/
+.idea
+*.iml
+
```
Comment (milismsft) [inline Search/BingImageSearch/multiFeatured/.gitignore]: it will be great if you can add the two file extensions that should not be included in this PR to be skipped as well by git

File: Search/BingImageSearch/multiFeatured/src/main/java/com/microsoft/azure/cognitiveservices/search/imagesearch/samples/BingImageSearchSample.java [hunk 6]
```diff
@@ -293,7 +293,7 @@ public static void main(String[] args) {
             //=============================================================
             // Authenticate
 
-            final String subscriptionKey = System.getenv("AZURE_BING_SAMPLES_API_KEY");
+            final String subscriptionKey = "bbfb78ad189649248750011e83b7b22d";//System.getenv("AZURE_BING_SAMPLES_API_KEY");
```
Comment (milismsft) [inline Search/BingImageSearch/multiFeatured/src/main/java/com/microsoft/azure/cognitiveservices/search/imagesearch/samples/BingImageSearchSample.java]: This is totally wrong, we don't use plain keys in the samples! The key should be read from the env var. Please make sure the corresponding resource was deleted to avoid any unauthorized usage.

Comment (aahill) [reply Search/BingImageSearch/multiFeatured/src/main/java/com/microsoft/azure/cognitiveservices/search/imagesearch/samples/BingImageSearchSample.java]: Ack, can't believe I missed that. Apologies - Resource has been deleted.

**Comment by milismsft** [comment]
LGTM I will merge the PR once the local files have been removed from the PR.

**Comment by aahill** [comment]
The local files should be removed now. Thanks for the review!
